### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by LounaCodes** [rbwhitaker/rbwhitaker.wikidot.com#10] [Ideas]
Hey, i really like your article about monogame, but since it's a lot of reading, It can be quite taxing on the eyes after a while with that white background. It would maybe help to add some sort of dark mode switch or option. Thanks

**Comment by rbwhitaker** [comment]
Alas, I'm somewhat limited to what Wikidot offers. I can provide CSS for the site, but cannot add a feature like the ability to choose which theme from several to use. There may come a day when I put the content on a website of my own control--it's something I've been thinking about for a long while--but until that happens, I don't think I have a lot of wiggle room here.
